clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -91,6 +91,16 @@ Improvements to clang-tidy
 New checks
 ^^^^^^^^^^
 
+- New :doc:`bugprone-true-macro
+  <clang-tidy/checks/bugprone/true-macro>` check.
+
+  In C++, ``true`` is considered a keyword by the preprocessor so an ``#if true``
+  enters the true branch, while in C, ``true`` is not treated as a special 
+  keyword by the preprocessor, so the false branch is entered. 
+  
+  The check identifies such cases, when ``true`` is used without being defined
```

Comment:
Once sentence is enough. See previous release Release Notes as example of such sentences.